src/main/java/org/ow2/proactive/catalog/report/CellFactory.java

```diff
@@ -0,0 +1,138 @@
+import java.net.URL;
+
+import javax.imageio.ImageIO;
+
+import org.apache.pdfbox.pdmodel.PDPage;
+import org.springframework.stereotype.Component;
+
+import be.quodlibet.boxable.Cell;
+import be.quodlibet.boxable.HorizontalAlignment;
+import be.quodlibet.boxable.Row;
+import be.quodlibet.boxable.VerticalAlignment;
+import be.quodlibet.boxable.image.Image;
+
+
+@Component
+public class CellFactory {
+
+    private static final int SMALL_FONT = 4;
+
+    private static final int MEDIUM_FONT = 6;
+
+    private static final int BIG_FONT = 8;
+
+    private static final int MAIN_TITLE_FONT_SIZE = 20;
+
+    private static final String ACTIVEEON_BLUE = "#0E2C65";
+
+    private static final String ACTIVEEON_ORANGE = "#EE7939";
+
+    private static final String LIGHT_GRAY = "#D3D3D3";
+
+    private static final String LIGHT_CYAN = "#F3F3F4";
+
+    private static final String WHITE = "#ffffff";
+
+    private static final String BLACK = "#000000";
+
+    public void addMainTitleCell(Row<PDPage> row, String data) {
+        createDataCell(row,
+                       (100 / 12f) * 9,
+                       data,
+                       MAIN_TITLE_FONT_SIZE,
+                       HorizontalAlignment.CENTER,
+                       VerticalAlignment.MIDDLE,
+                       ACTIVEEON_ORANGE,
+                       ACTIVEEON_BLUE);
+    }
+
+    public void addSecondaryHeaderCell(Row<PDPage> row, String data) {
+        createDataCell(row, 100, data, HorizontalAlignment.CENTER);
+    }
+
+    public void createDataCell(Row<PDPage> row, float width, String data) {
+        createDataCell(row, width, data, MEDIUM_FONT);
+    }
+
+    public void createDataHeaderCell(Row<PDPage> row, float width, String title) {
+        createDataCell(row,
+                       width,
+                       title,
+                       BIG_FONT,
+                       HorizontalAlignment.CENTER,
+                       VerticalAlignment.TOP,
+                       LIGHT_CYAN,
+                       BLACK);
+    }
+
+    public void createDataCellBucketName(Row<PDPage> row, float width, String data) {
+        createDataCell(row, width, data, BIG_FONT, HorizontalAlignment.CENTER, VerticalAlignment.MIDDLE, WHITE, BLACK);
+    }
+
+    public void createKeyValueContentDataCell(Row<PDPage> row, float width, String data) {
+        createDataCell(row, width, data, SMALL_FONT, HorizontalAlignment.LEFT, VerticalAlignment.TOP, WHITE, BLACK);
+    }
+
+    public void createDataCell(Row<PDPage> row, float width, String data, int fontSize) {
+        createDataCell(row, width, data, fontSize, HorizontalAlignment.CENTER, VerticalAlignment.TOP, WHITE, BLACK);
+    }
+
+    public void createDataCell(Row<PDPage> row, float width, String data, HorizontalAlignment align) {
+        createDataCell(row, width, data, MEDIUM_FONT, align, VerticalAlignment.TOP, WHITE, BLACK);
+    }
+
+    public void createIconCell(Row<PDPage> row, float width, String url_path) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)